### PR TITLE
No defaulting to inter Tuleva transfers/future contributions fund, al…

### PR DESCRIPTION
**Pull request by JordanValdma**
…so skip transfers from funds balances of zero

### CODE REVIEW

File: src/exchange/reducer.js [hunk 1]
```diff
@@ -66,7 +66,9 @@ const initialState = {
 
 function createFullDefaultSourceSelection({ sourceFunds, targetFunds }) {
   return sourceFunds
-    .filter(fund => fund.isin !== targetFunds[0].isin)
+  // .filter(fund => fund.isin !== targetFunds[0].isin)
```
Comment (ErkoRisthein) [inline src/exchange/reducer.js]: comment 👮

Comment (ErkoRisthein) [inline src/exchange/reducer.js]: Is there a test case too? 👮

Comment (JordanValdma) [inline src/exchange/reducer.js]: Already fixed the comment in master, and yes there's a test case, read the test

Comment (ErkoRisthein) [inline src/exchange/reducer.js]: 💪

**Pull request merged**